homeassistant/components/qbittorrent/helpers.py

```diff
@@ -2,10 +2,12 @@
 from qbittorrent.client import Client
 
 
-def setup_client(url: str, username: str, password: str, verify_ssl: bool) -> Client:
+def setup_client(
+    url: str, username: str, password: str, verify_ssl: bool
+) -> tuple[Client, bool]:
```

Comment:
Should we make this a defined data structure to plan for more data being passed down in the future? Along with it being clearer what the bool is currently for.

I'm not opposed to leaving as a tuple for now, just want to put the idea out there if you want to swap it to a defined data structure

Reply:
Sound more interesting for future additions, so creating a dataclass in there ?

Reply:
I'll check this, I will maybe wait for the change of lib so we can maybe use the lib type in this :+1: